mito-ai/mito_ai/completions/handlers.py

```diff
@@ -222,7 +222,34 @@ async def on_message(self, message: str) -> None: # type: ignore
                 )
                 self.reply(reply)
             return
-        
+
+        if type == MessageType.STOP_AGENT:
+            # Get the thread_id from metadata if available, otherwise use the newest thread
```

Comment:
This is out of date. If the thread_id is not passed, then we don't process the request